### ISSUE TITLE
Becomes unusably slow

### ISSUE THREAD

**Issue by PassiveLemon** [leinelissen/jellyfin-audio-player#122]
This appears to be Fintunes in the appstore just based by the pictures.

Using it for more than 10 seconds results in it becoming unuseably slow. The UI takes minutes to change tabs and respond to inputs. This appears to happen when downloading. A 30 song playlist took about 10 minutes to resolve but trying to download my around 200 songs playlist results in the app becoming permanently slow and not functional. I am not sure what causes this and I haven't seen any other reports.

Overall the app is great and way nicer than Finamp but because of this, I have to continue to use Finamp.

**Comment by leinelissen** [comment]
Yes, you're right, thanks for making an issue! This is an interesting problem. I'll try and reproduce it, but I haven't had any issues in this domain yet. In the meantime, could you share what device you're using so I can attempt to replicate? Cheers!

**Comment by PassiveLemon** [comment]
I am on an iPhone 12 Pro 15.2. It's possible the iOS version has some issues but I am staying for the jailbreaks.

**Comment by leinelissen** [comment]
Okay I managed to replicate the issue. No worries on leaving though, I still think it's a relevant issue.

**Comment by dbolger** [comment]
I'm also seeing occasional slowdowns when using the app. Specifically, when I download an entire album and try to navigate between screens, the UI begins to lock up.

**Comment by alanorth** [comment]
I notice this on Android as well. When I select "download album" the interface becomes unusably slow.

**Comment by PassiveLemon** [comment]
Closing stale issue
